src/Oscoin/Consensus/Validation.hs

```diff
@@ -2,21 +2,36 @@
 -- sophisticated ones.
 -}
 module Oscoin.Consensus.Validation
-    ( validateParentHash
+    ( validateHeight
+    , validateParentHash
     , validateDifficulty
     , validateTimestamp
     ) where
 
 import           Oscoin.Prelude
 
 import           Oscoin.Consensus.Types
-import           Oscoin.Crypto.Blockchain
+import           Oscoin.Crypto.Blockchain hiding (height)
 import           Oscoin.Crypto.Blockchain.Block (Difficulty(..))
 import           Oscoin.Crypto.Hash (Hash)
 import           Oscoin.Time
 
 import           Control.Monad.Except (Except, throwError)
 
+-- | Rejects a block if its parent's height is smaller or equal than the
+-- input one.
+validateHeight
+    :: Block c tx s
+    -> Block c tx s
+    -> Except (ValidationError c) ()
+validateHeight parent blk
+    | height <- blockHeight (blockHeader blk)
+    , height <= parentHeight =
```

Comment:
This would allow heights to be skipped - should be `height == succ parentHeight`

Reply:
Oops, yes -- this is what happens when you try to cram to many PRs in one day 😉 Sorry 'bout that!